flexmeasures/data/schemas/sensors.py

```diff
@@ -158,6 +158,8 @@ def validate_generic_asset(self, generic_asset_id: int):
             raise ValidationError(
                 f"Generic asset with id {generic_asset_id} doesn't exist."
             )
+        # Add it to context to use it for AssetAuditLog record
+        self.context["generic_asset"] = generic_asset
```

Comment:
Can you explain or give me a link so I can read more about this context? It seems I don't know about this.

Reply:
Sure. From docs: "The context attribute of a Schema is a general-purpose store for extra information that may be needed for (de)serialization."
https://marshmallow.readthedocs.io/en/stable/extending.html